readme.md

```diff
@@ -38,10 +41,29 @@ The source object to copy properties from.
 
 #### mapper
 
-Type: `(sourceKey, sourceValue, source) => [targetKey, targetValue, mapperOptions?] | mapObjectSkip`
+Type: `(sourceKey, sourceValue, source, path) => [targetKey, targetValue, mapperOptions?] | mapObjectSkip`
 
 A mapping function.
 
+##### path
+
+Type: `string[]`
+
+When using `deep: true`, this is the sequence of keys to reach the current value from the `source`, otherwise it is an empty array.
+
+For arrays, the key is the index of the element being mapped.
+
+```js
```

Comment:
Let me know if this simplified example is ok, in response to https://github.com/sindresorhus/map-obj/pull/43/files#r884235030